mod-source-record-manager-server/src/main/java/org/folio/rest/impl/ChangeManagerImpl.java

```diff
@@ -204,7 +206,14 @@ public void postChangeManagerJobExecutionsRecordsById(String id, boolean acceptI
         eventDrivenChunkProcessingService.processChunk(entity, id, acceptInstanceId, params)
           .map(processed -> PostChangeManagerJobExecutionsRecordsByIdResponse.respond204())
           .map(Response.class::cast)
-          .otherwise(ExceptionHelper::mapExceptionToResponse)
+          .otherwise(ex -> {
+            if (ex instanceof DuplicateEventException) {
+              LOGGER.warn("postChangeManagerJobExecutionsRecordsById:: Failed to process chunk of RawRecords with JobExecution id {}: {}", id, ex.getMessage());
+              return ExceptionHelper.mapExceptionToResponse(new BadRequestException(ex.getMessage()));
+            } else {
+              return ExceptionHelper.mapExceptionToResponse(ex);
+            }
+          })
           .onComplete(asyncResultHandler);
       } catch (Exception e) {
         LOGGER.warn(getMessage("postChangeManagerJobExecutionsRecordsById:: Failed to process chunk of RawRecords with JobExecution id {}", e, id));
```

Comment:
I think that the order for "e" and "id" is incorrect.

Reply:
@Aliaksandr-Fedasiuk No, you are not correct. Please look at how arguments inside the method "getMessage()" are provided.

Reply:
Hmm. Thanks, but are we sure we need to use the getMessage method?

Reply:
@Aliaksandr-Fedasiuk, I hope this is needed for formatting logs with an exception. This is used in all endpoints in this module. Let's leave it here.

Reply:
Thanks